### ISSUE TITLE
i can' open the app on ubuntu

### ISSUE THREAD

**Issue by NOOBPLAYER8115** [benjamimgois/goverlay#45]
every time i want to open goverlay it does nothing.

**Comment by Gawrion** [comment]
the same here. Running from ubuntu discover doeas nothing. Running from console gives me "can't get libdl.so"

**Comment by benjamimgois** [comment]
I think the major problem with goverlay on ubuntu based distros isn't goverlay it self, but the old mangohud version on debian repositories. Goverlay start script force the use of mangohud with it. Please, extract the tar.xz version of goverlay below and run the goverlay binary, check if it work.

https://github.com/benjamimgois/goverlay/releases/download/0.9.1/goverlay_0_9_1.tar.xz

**Comment by Gawrion** [comment]
Now i get this: ./goverlay: symbol lookup error: ./goverlay: undefined symbol: QLCLOpenGLWidget_Create

**Comment by benjamimgois** [comment]
Check if you have qt5pas

`sudo apt install libqt5pas-dev`

and try again

**Comment by Gawrion** [comment]
I have it. The same error. Finally i got vkbasalt working standalone, so my motivation is lowered, but it will be nice to have goverlay working too. I tried to add and build some dependencies manually and now, after running goverlay in console i got: aborted (core dumped) :/ So i think i made progress with this qt5pas error and now something else is blocking :/

**Comment by Gawrion** [comment]
UPDATE: now its working :D I did so many things, so i even dont know what helped. BTW i have mangohud 6.5.

**Comment by benjamimgois** [comment]
glad that it worked  ! i still have to do a full review of the ubuntu install procedure. Will try to review it for next release.

**Comment by Gawrion** [comment]
I'm sorry to write this but I was happy to fast... Idk what i did it run, but now it wont run again but i haven't changed anything :/ When it run i had some plasma crash report i opened it, clicked again on run and it worked. Then i closed and never succeed again :/ 

If u want some testing in the future just tell me ;)

**Comment by Yahiko777** [comment]
I'm getting the same issue. I have built mangohud manually per the readme file, extracted and ran the tar.xz file you have listed here, and ran the apt command listed prior, and I still get "can't get libdl.so" when trying to run GOverlay. I'm very confused as I have found no other methods online as to how to get this application to run. I'm running Ubuntu 20.04.

**Comment by Gawrion** [comment]
Hej benjamimgois. I have some issue. I uninstalled goverlay, mangohud, and left only vkbasalt, but my settings from one goverlay succesfull run are stil valid somewhere. Where goverlay keeps setting so i can edit it? Can u give me some hints?

**Comment by Gawrion** [comment]
Ok. I got it: /home/user/.config/vkBasalt/vkBasalt.conf
May be usefull for others.

ps. my problem was Dolphin doesnt search in hidden folders. I work on slackware >30 years, but only command line. Now i installed Kubuntu and it seems i got to learn many things from scratch :)

**Comment by Gawrion** [comment]
Hey Benjamin! I just want u to know - i installed bare Nobara (Fedora based). Goverlay is preinstalled there but weird things happen too. It is better than ubuntu - i can open Goverlay every time but... First time mangohud worked, than i clicked some menu options and it dissapeared... Reboot - the same... dnf reinstall mangohud/goverlay - no help. I got some issueas with vkBasalt plugin too. I have some effects enabled, but they dont show on active effects tab.  And last thing - testing vkBasalt on this fast circling box is very hard. It has lack of colors, and circles round to fast for some effects to see. It should have more pallete colours and move slower.

EDIT: i changed quick layouts on extras tab and mango reapeared.

**Comment by Romannoss** [comment]
i'm using kubuntu 22.04 LTS and happening the same error "can't get libdl.so"  :(

**Comment by AdivonSlav** [comment]
The libqt5pas-dev package needed to compile goverlay manually is outdated from the Ubuntu PPA's if using an Ubuntu-based distro. Install the packages manually from the latest release at https://github.com/davidbannon/libqt5pas/releases and then goverlay should succesfully compile with make

Also see this https://github.com/benjamimgois/goverlay/issues/53#issuecomment-1537548338

**Comment by benjamimgois** [comment]
Yeah, ubuntu based distros have being very tought to support due to older packages. Right now i'm finishing the latest touches in the new UI for the next release. Hope the newer code will fix these long standing issues.

**Comment by woeisme** [comment]
> Yeah, ubuntu based distros have being very tought to support due to older packages. Right now i'm finishing the latest touches in the new UI for the next release. Hope the newer code will fix these long standing issues.

please please please release a flatpak version when you do.
